Riot/Modules/Room/VoiceMessages/VoiceMessageToolbarView.swift

```diff
@@ -130,6 +131,8 @@ class VoiceMessageToolbarView: PassthroughView, NibLoadable, Themable, UIGesture
         playbackView.waveformView.addGestureRecognizer(tapGesture)
         tapGesture.delegate = self
         self.tapGesture = tapGesture
+        primaryRecordButton.accessibilityLabel = NSLocalizedString("room_accessibility_record_voice_message", tableName: "Vector", comment: "A11y label for the Record Voice Message button.")
+        primaryRecordButton.accessibilityHint = NSLocalizedString("room_accessibility_record_voice_message_hint", tableName: "Vector", comment: "A11y hint for the Record Voice Message button.")
```

Comment:
One small change you can make is to use the strings generated in `Riot/Generated/Strings.swift`. They're accessible through `VectorL10n` after you compile (for both Swift and objc), so you could do the following:

```suggestion
        primaryRecordButton.accessibilityLabel = VectorL10n.roomAccessibilityRecordVoiceMessage
        primaryRecordButton.accessibilityHint = VectorL10n.roomAccessibilityRecordVoiceMessageHint
```